README.md

```diff
@@ -334,10 +335,10 @@ which developers will be able to change to customize look and feel.
 - `display: none`: this will keep the toast hidden from view and searching
 when the `open` attribute is not present.
 
-#### `std-toast[open]`
+#### `std-toast([open])`
```

Comment:
```suggestion
#### `std-toast[open]`
```

Looks like you didn't rebase on my typo fix from last night.

Reply:
My bad, I must have missed it